components/global-header/GlobalHeader.tsx

```diff
@@ -50,6 +55,27 @@ export const GlobalHeader = ({}: IProps) => {
               label="Settings"
               icon={EIConTypes.SETTINGS}
               name="settings"
+              onCustomClick={() => {
+                dispatchModal({
+                  type: EModalActions.SET_CONTENT,
+                  payload: {
+                    title: 'Settings',
+                    content: (
+                      <UserSettings
+                        onClose={closeModal}
+                        onSave={(options) => {
+                          toast(
+                            'Saved! Sorry. No apis for this at the moment ¯\\_(ツ)_/¯',
```

Comment:
If the api is not available how can the settings be saved?

Reply:
Or are the user settings only saved in the cache (context) of the app?

Reply:
Nothing is saved. only an alert is triggered. If there is no apis to persists the data, i am not saving it on context.